platform/ios/PolyPodApp/PolyPod/FeatureView.swift

```diff
@@ -137,7 +138,7 @@ struct FeatureView: View {
             preferredStyle: UIAlertController.Style.alert)
         alert.addAction(UIAlertAction(
                             title: NSLocalizedString(
-                                "button_url_open_confirm",
+                                "button_confirm",
```

Comment:
Isn't this too general? Maybe `button_endpoint_confirm`?

Reply:
As it is yes, I actually went for something generic on purpose